### PR TITLE
sncosmo 1.4.0, extinction 0.2.2

### PR DESCRIPTION
**Pull request by kbarbary**
cython module that used to be in sncosmo has been factored out into a separate module called extinction. Not sure if I need a custom recipe for extinction...

### CODE REVIEW

**Comment by mwcraig** [comment]
@bsipocz -- is it ok to close #94?

**Comment by kbarbary** [comment]
Didn't see that one!

**Comment by bsipocz** [comment]
Sure, I've closed it.

**Comment by kbarbary** [comment]
Hmmm, do I need to add `extinction` as a build-time dependency of sncosmo?

Also, I put `extinction` on conda-forge. Is that messing anything up? Can I remove the `numpy_compiled_extensions` line in `extinction` here?

File: recipe_templates/sncosmo/meta.yaml [hunk 1]
```diff
@@ -37,13 +37,13 @@ requirements:
     - scipy
     - astropy
     - astropy-helpers  # If this ever gets submoduled in here this recipe can go
-    - cython # There is now one cython file included.
```
Comment (mwcraig) [inline recipe_templates/sncosmo/meta.yaml:40]: Looks like you need extinction here too.

File: requirements.yml [hunk 2]
```diff
@@ -224,3 +224,7 @@
 
 - name: pytest-mpl
   version: 0.5
+
+- name: extinction
+  version: 0.2.2
+  numpy_compiled_extensions: true
```
Comment (mwcraig) [inline requirements.yml]: This can be removed.

File: requirements.yml [hunk 3]
```diff
@@ -224,3 +224,7 @@
 
 - name: pytest-mpl
   version: 0.5
+
+- name: extinction
```
Comment (mwcraig) [inline requirements.yml:228]: This _should_ be copied from conda-forge without any additional action on your part....the way the recipes are generated right now is:
- Use a recipe template if there is one.
- Copy the recipe from conda-forge if it exists.
- Generate a recipe using `conda skeleton`

If it exists the package is on the conda-forge channel the binary gets copied into the astropy channel without building.

**Review by mwcraig — COMMENTED** [review]
I think a couple minor changes should get this working.

File: requirements.yml (hunk 3 above)
Comment (kbarbary) [reply requirements.yml:228]: Thanks for the detailed explanation Matt!

**Comment by mwcraig** [comment]
Merging, thanks @kbarbary

**Pull request merged**